### ISSUE TITLE
Published figures and projects description

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-viewer-fiji#781]
Right now the published projects and figures are backed into the source code.
This is really bad for updating them (because it needs interaction with the Fiji MoBIE update site), especially when external users would like to register their projects after publication.

What about creating an extra repo only containing JSON files that contain this information? This would be much easier to update and we could grant push rights to many more people.

The Fiji commands would then download these JSON files to build the selection UI.

**Comment by constantinpape** [comment]
Yes, I agree. But I wouldn't create an extra repo for it, and instead just save it on https://github.com/mobie/mobie.github.io.
Do you have any preferences for the json name and layout?

**Comment by tischi** [comment]
> Do you have any preferences for the json name and layout?

We could simply serialize the classes:

1. https://github.com/mobie/mobie-viewer-fiji/blob/main/src/main/java/org/embl/mobie/viewer/project/PublishedFigure.java
2. https://github.com/mobie/mobie-viewer-fiji/blob/main/src/main/java/org/embl/mobie/viewer/project/PublishedProject.java

But we may still adapt the names of the fields in those classes in the Java code to make it as similar as possible to the menu entries in Fiji.
